common/models/user.js

```diff
@@ -255,6 +261,7 @@ module.exports = function(User) {
               err = new Error(g.f('login failed as the email has not been verified'));
               err.statusCode = 401;
               err.code = 'LOGIN_FAILED_EMAIL_NOT_VERIFIED';
+              err.userId = user.id;
```

Comment:
In production mode, strong-error-handler discards most error properties and keeps only `code` and `details`. As a result, the client won't be access `err.userId`.

My proposal:

```js
err.details = {
  userId: user.id,
};
```